sdk/java-sdk-spring/src/main/scala/kalix/javasdk/client/ComponentCall.scala

```diff
@@ -73,6 +105,19 @@ object ComponentCall {
     invoke(params, kalixClient, MethodRefResolver.resolveMethodRef(lambda), ids)
   }
 
+  private[client] def invoke[R](
```

Comment:
The old `invoke` method (just above this one) is not being used anymore. It can be removed.